src/ui/State.tsx

```diff
@@ -456,7 +456,7 @@ export class StateStore
       this.props.player.tracks.sort(
         (a, b) => Number(b.height) - Number(a.height),
       ),
-      'height',
+      'bandwidth',
```

Comment:
I believe we can drop the uniqBy filter above, as this would select "bandwidth" to group by.